### PR TITLE
Projects API Preview

### PR DESCRIPTION
**Pull request by balintant**
https://developer.github.com/changes/2016-09-14-projects-api/
https://developer.github.com/v3/repos/projects/

### CODE REVIEW

**Comment by shurcooL** [comment]
@balintant, we have an open issue tracking this new API addition at https://github.com/google/go-github/issues/432. Have you seen it?

@gmlewis self-assigned himself on that issue [9 hours ago](https://github.com/google/go-github/issues/432#event-800494176) to indicate he has started working on it, and I can see he has made a WIP implementation on [`add-projects-api` branch](https://github.com/google/go-github/compare/master...gmlewis:add-projects-api) of his fork.

You should coordinate with @gmlewis to avoid duplicate work on this.

**Comment by balintant** [comment]
@shurcooL nice 😃 I started this like 13 hours ago, but I had some stuff to do. So I just finished it as I arrived back home.

**Comment by balintant** [comment]
@shurcooL After reviewing that branch, I am closing this issue and letting him go with it. Thanks!

**Comment by gmlewis** [comment]
Thank you, @balintant for all the work you put into this PR... believe me, I know what kind of time commitment it takes, and your work is greatly appreciated!  I'm sorry for the conflicting PRs.
I look forward to seeing more contributions from you in the future!

**Comment by balintant** [comment]
I will add an issue and assign it to myself next time. 😊 Thanks

File: github/repos_projects.go [hunk 1]
```diff
@@ -0,0 +1,496 @@
...
+
+import "fmt"
+
+// Projects represents a Project's configuration of a GitHub Repository
+type RepositoryProject struct {
+	ID        *int       `json:"id,omitempty"`
+	Name      *string    `json:"name,omitempty"`
+	Body      *string    `json:"body,omitempty"`
+	URL       *string    `json:"url,omitempty"`
+	OwnerURL  *string    `json:"url,omitempty"`
```
Comment (shurcooL) [inline github/repos_projects.go]: This should be `json:"owner_url,omitempty"`, not `json:"url,omitempty"`.

**Pull request closed without merging**